From ecashin@coraid.com Thu Jan  5 08:42:05 2006
From: Ed L Cashin <ecashin@coraid.com>
To: Coywolf Qi Hunt <coywolf@gmail.com>
Cc: Greg K-H <greg@kroah.com>
Subject: aoe: use less confusing driver name
Date: Thu, 05 Jan 2006 11:35:29 -0500
Message-ID: <87hd8i63am.fsf@coraid.com>

Users were confused by the driver being called "aoe-2.6-$version".
This form looks less like a Linux kernel version number.

Signed-off-by: "Ed L. Cashin" <ecashin@coraid.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/block/aoe/aoemain.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/block/aoe/aoemain.c
+++ gregkh-2.6/drivers/block/aoe/aoemain.c
@@ -89,7 +89,7 @@ aoe_init(void)
 	}
 
 	printk(KERN_INFO
-	       "aoe: aoe_init: AoE v2.6-%s initialised.\n",
+	       "aoe: aoe_init: AoE v%s initialised.\n",
 	       VERSION);
 	discover_timer(TINIT);
 	return 0;
